types/lodash/fp.d.ts

```diff
@@ -1984,7 +1984,7 @@ declare namespace _ {
     type LodashIsNil = (value: any) => value is null | undefined;
     type LodashIsNull = (value: any) => value is null;
     type LodashIsNumber = (value: any) => value is number;
-    type LodashIsObject = (value: any) => boolean;
+    type LodashIsObject = (value: any) => value is object;
```

Comment:
`fp.d.ts` is auto-generated, so don't edit it directly. Instead, edit `common/lang.d.ts`, then do `npm run generate` inside of the `scripts` folder (see `readmen.md` in the lodash folder)

Reply:
Done, thanks